MathlibTest/CountHeartbeats.lean

```diff
@@ -19,3 +19,33 @@ example (a : Nat) : a = a := rfl
 
 guard_min_heartbeats 1 in
 example (a : Nat) : a = a := rfl
+
+/-!
+# Tests for the `countHeartbeats` linter
+-/
+
+set_option linter.countHeartbeats true
+
+-- sets the `countHeartbeats` linter option to `true`
+#count_heartbeats
```

Comment:
I am a bit confused: do I understand correctly that both `set_option linter.countHeartbeats true` and `#count_heartbeats` do the same? Maybe we should have two different sections in the test file, so they run individually?

Reply:
Your understanding is correct: there is a discussion/poll [here](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/count_heartbeats.20for.20all.20declarations.20in.20a.20file.3F/near/492959598) about the appropriateness of the two spellings.

I also now used two sections for testing the two commands.